services/slack/intermediate.go

```diff
@@ -117,14 +117,28 @@ func (t *Transformer) TransformUsers(users []SlackUser) {
 
 	resultUsers := map[string]*IntermediateUser{}
 	for _, user := range users {
-		newUser := &IntermediateUser{
-			Id:        user.Id,
-			Username:  user.Username,
-			FirstName: user.Profile.FirstName,
-			LastName:  user.Profile.LastName,
-			Position:  user.Profile.Title,
-			Email:     user.Profile.Email,
-			Password:  model.NewId(),
+		var newUser *IntermediateUser
+
+		if user.IsBot {
+			newUser = &IntermediateUser{
```

Comment:
All the other fields are same for is only change botID no need to create the whole user 2 times